### ISSUE TITLE
PHP Deprecation warning: ord(null)

### ISSUE THREAD

**Issue by getdatakick** [mrclay/jsmin-php#16]
Reproduced in version: `2.4.1` - looks like fix for #10 was incomplete. 

When input string ends with single line comment PHP raises deprecation warning:

`ord(): Passing null to parameter #1 ($character) of type string is deprecated`

at

`vendor/mrclay/jsmin-php/src/JSMin/JSMin.php line 390`

#### Reprosteps:

This problem occurs when input string ends with single line comments. Simplest test case: 

```php
\JSMin\JSMin::minify('// comment');
```

#### Stacktrace:
```
#1  vendor/mrclay/jsmin-php/src/JSMin/JSMin.php(390): ord(null)
#2  vendor/mrclay/jsmin-php/src/JSMin/JSMin.php(446): JSMin\JSMin->consumeSingleLineComment()
#3  vendor/mrclay/jsmin-php/src/JSMin/JSMin.php(230): JSMin\JSMin->next()
#4  vendor/mrclay/jsmin-php/src/JSMin/JSMin.php(121): JSMin\JSMin->action(3)
#5  vendor/mrclay/jsmin-php/src/JSMin/JSMin.php(86): JSMin\JSMin->min()
#6  JSMin\JSMin::minify("// comment")
```

**Comment by glensc** [comment]
Fixed via #9

**Comment by glensc** [comment]
Note to self, test was added in 4d92a91a7feea765cbc1c5bc5ad8184fa09a1406
